packages/ui/src/components/Form/ExpandableDetails.tsx

```diff
@@ -18,6 +18,8 @@ export const ExpandableDetails: FunctionComponent<PropsWithChildren<ExpandableDe
 
       <ExpandableSection
         toggleText={isExpanded ? 'Hide properties' : 'Show properties'}
+        toggleId="expandable-section-toggle"
+        contentId="expandable-section-content"
```

Comment:
This is necessary because without both properties set, the id gets generated by a timestamp, breaking the snapshot tests on every run.